client/layout/index.jsx

```diff
@@ -307,9 +307,7 @@ class Layout extends Component {
 			'is-unified-site-sidebar-visible': this.props.isUnifiedSiteSidebarVisible,
 			'is-blaze-pro': this.props.isBlazePro,
 			'is-woo-com-oauth': isWooOAuth2Client( this.props.oauth2Client ),
-			'feature-flag-woocommerce-core-profiler-passwordless-auth': config.isEnabled(
-				'woocommerce/core-profiler-passwordless-auth'
-			),
+			'feature-flag-woocommerce-core-profiler-passwordless-auth': true,
```

Comment:
I've noticed that this class is being applied to pretty much every screen in calypso 😅 and it must be coming from here. Shouldn't there be a condition here to only apply it in the woo passwordless auth screen? And probably in the same in the other two instances where it's being applied.

Reply:
The plan was to refactor the woo.scss file so we don't need this class name at all. 😓  Yeah, that makes sense. We should only add the class name for Woo.